### DISCUSSION TITLE
Using External SMTP

### DISCUSSION THREAD

**Discussion by rajeevrrs** [modoboa/modoboa#2662] [General]
Hi, 

1. How to use external SMTP (Amazon SES) for sending a mail. (How to config external SMTP in Modoboa Webmail or Modoboa Admin Panel)
2. How to install Roudcube for webmail client?

**Comment by tonioo** [comment]
To use an external SMTP server, you need to update postfix configuration directly, there is no way to do it from Modoboa.
And about Roundcube, just follow their regular install procedure.

**Reply by rajeevrrs** [reply]
Any guide how to do that in postfix. 

I have installed it but after successfully installation my server always crash or mail.mydomain.com never low.
